lib/files.js

```diff
@@ -0,0 +1,30 @@
+const fs = require('fs')
+const path = require('path')
+
+const readJson = filename => {
+	return new Promise((resolve) => {
+		fs.readFile(filename, 'utf8', function (err, data) {
+			if (err) throw err
+			resolve(JSON.parse(data))
+		})
+	})
+}
+
+const writeJson = (filename, json) => {
+	return new Promise((resolve) => {
+		fs.writeFile(filename, json, (err) => {
+			if (err) throw err
+			resolve()
+		})
+	})
+}
+
+const folderJsonFiles = (dir) => {
```

Comment:
Please rename to something like `listJsonFileNames`. I found previous name not very descriptive.